src/client/main.ts

```diff
@@ -232,21 +231,25 @@ if (channelPromise) {
 /**
  * Adds a listener for incoming Intents from the Agent.
  */
-export function addIntentListener(intent: string, handler: (context: Context) => void): IntentListener {
+export function addIntentListener(intent: string, handler: (context: Context) => any): IntentListener {
```

Comment:
Just want to double check the change from void => any is what we want

Reply:
Actually, that change to the spec is still in flight, so good point, I reverted to `void`.  This type will need to change, but later.